Resources/Prototypes/_Carpmosia/Maps/feint.yml

```
@@ -0,0 +1,65 @@
+- type: gameMap
+  id: Feint
+  mapName: 'Feint'
+  mapPath: /Maps/_Carpmosia/feint.yml
+  minPlayers: 10
+  maxPlayers: 35 
+  stations:
+    Feint:
+      stationProto: StandardNanotrasenStation
+      components:
+      - type: StationNameSetup
+        mapNameTemplate: '{0} Feint Station {1}'
+        nameGenerator:
+          !type:NanotrasenNameGenerator
+          prefixCreator: '-TE'
+      - type: StationEmergencyShuttle
+        emergencyShuttlePath: /Maps/_Carpmosia/Shuttles/emergency_feint.yml
+      - type: StationJobs
+        availableJobs: # 43 jobs total w/o latejoins & interns, 58 jobs total w/ latejoins & interns
```

```suggestion
        availableJobs: # 44 jobs total w/o latejoins & interns, 59 jobs total w/ latejoins & interns
```
hhhhhhhhhhhhhhhhhhhhhh